utils/rpms/daos.spec

```diff
@@ -553,12 +549,16 @@ getent passwd daos_agent >/dev/null || useradd -s /sbin/nologin -r -g daos_agent
 # No files in a shim package
 
 %changelog
+* Wed Mar 29 2023 Ryon Jensen <ryon.jensen@intel.com> 2.3.106-3
+- Removed unnecessary test files
+
 * Fri Mar 17 2023 Tom Nabarro <tom.nabarro@intel.com> 2.3.106-2
 - Add numactl requires for server package
 
 * Tue Mar 14 2023 Brian J. Murrell <brian.murrell@intel.com> 2.3.106-1
 - Bump version to be higher than TB5
 
+
```

Comment:
Looks like you inadvertently added a blank line here.  It shouldn't be fatal but if you have to push anything else to this PR, please remove this.